### PR TITLE
Add jupyter-nbmodel-client

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by fcollonval — COMMENTED** [review]
@echarles, could you comment you accept to be added as maintainer of this recipe?

**Comment by echarles** [comment]
> @echarles, could you comment you accept to be added as maintainer of this recipe?

Sure, I agree.

**Review by echarles — APPROVED** [review]
LGTM Thx @fcollonval

**Comment by fcollonval** [comment]
@conda-forge/help-python, ready for review!

File: recipes/jupyter-nbmodel-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (dbast) [inline recipes/jupyter-nbmodel-client/meta.yaml:7]:
```suggestion
```

File: recipes/jupyter-nbmodel-client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 1cc666634b92b7375fd5e3f62a96fc9eb0d8ab05fcbb2b6c1be4081878af2c7a
+  
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/jupyter-nbmodel-client/meta.yaml:21]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```

**Comment by fcollonval** [comment]
Thanks for the review @dbast and @ocefpaf. I applied the suggestions. Would you mind having a second look?

**Pull request merged**